app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/ShowTicketActivity.java

```diff
@@ -69,9 +66,11 @@ public void onCreate(Bundle savedInstanceState) {
         String String = event.getTitle();
         String eventLocationString = event.getLocality();
         String eventDateTimeString = dateString + " " + time[1];
-        String eventPriceString = String.format(Locale.GERMANY, "%.2f",
-                ticketType.getPrice()) + " €";
-
+        String eventPriceString = "";
+        if(ticketType != null) {
+            eventPriceString = String.format(Locale.GERMANY, "%.2f",
+                    ticketType.getPrice() / 100) + " €";
```

Comment:
Centralize this formatting somewhere. This is already the second time I've seen this. Also please do this properly, not everybody is from Germany.

Reply:
I put a method into the TicketType class that returns the formated string we need.